v2/yarpcjson/inbound_test.go

```diff
@@ -101,13 +101,13 @@ func TestHandleInterfaceEmptySuccess(t *testing.T) {
 	handler := jsonHandler{reader: ifaceEmptyReader{}, handler: reflect.ValueOf(h)}
 
 	reqBuf := yarpc.NewBufferString(`["a", "b", "c"]`)
-	_, _, err := handler.Handle(context.Background(), &yarpc.Request{
+	_, respBuf, err := handler.Handle(context.Background(), &yarpc.Request{
```

Comment:
Did we decide once and for all to use req/resp throughout spring?

Reply:
Updated this for consistency on `req`/`res`

Reply:
Heroic.